core/text/code_formatter.talon-list

```diff
@@ -4,6 +4,8 @@ all cap: ALL_CAPS
 all down: ALL_LOWERCASE
 camel: PRIVATE_CAMEL_CASE
 dotted: DOT_SEPARATED
+listed: COMMA_SEPARATED
+args: COMMA_SEPARATED
```

Comment:
"args" is used in community to insert parentheses. We probably shouldn't overload this with another definition.